code/datums/diseases/advance/advance.dm

```diff
@@ -31,6 +30,7 @@
 	var/list/symptoms = list() // The symptoms of the disease.
 	var/id = ""
 	var/processing = FALSE
+	var/mutable = TRUE //set to FALSE to prevent most in-game methods of altering the disease via virology

```

Comment:
Is there any reason to have /advance subtype now ? As in the core diseases being static was the major difference from what i can remember.

Reply:
The /advance subtype has datum symptoms, while regular diseases do not.